gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/DnsExample.java

```diff
@@ -441,7 +441,8 @@ private static void printZone(Zone zone) {
     System.out.printf("%nName: %s%n", zone.name());
     System.out.printf("ID: %s%n", zone.generatedId());
     System.out.printf("Description: %s%n", zone.description());
-    System.out.printf("Created: %s%n", FORMATTER.format(new Date(zone.creationTimeMillis())));
+    DateFormat formatter = new SimpleDateFormat("YYYY-MM-dd HH:mm:ss");
+    System.out.printf("Created: %s%n", formatter.format(new Date(zone.creationTimeMillis())));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.out.printf is used](https://www.codacy.com/app/mziccard/gcloud-java/file/2351654449/issues/source?bid=3123103&fileBranchId=3217862#l445)